File: counting.go
package main

import (
	"fmt"
	"time"
)

func main() {
	var count int = 0
	startTime := time.Now().UnixNano() / 1000000
	for i := 1; i <= 1000000000; i++ {
		count++
	}
	endTime := time.Now().UnixNano() / 1000000
	fmt.Println("Count:", count);
	fmt.Println("Execution time:" , float64(endTime - startTime)/1000, " seconds")
}

File: counting.rs
use std::time::Instant;

fn main() {
    let start_time = Instant::now();
    let mut count = 0;
    for _ in 1..=1_000_000_000 {
        count += 1;
    }
    let end_time = Instant::now();
    println!("Count: {}", count);
    println!("Execution time: {} seconds", (end_time - start_time).as_secs_f64());
}